e2e/ansible/playbooks/hyperconverged/test-replica-node-affinity/replica-node-affinity.yml

```diff
@@ -164,7 +164,7 @@
          retries: 5
 
        - name: 8) Get pod name
-         shell: kubectl get pods -n "{{ namespace }}" -o wide | grep "{{ pv_name.stdout }}" | grep rep | grep "{{ node1.stdout }}" | awk '{print $1}'
+         shell: kubectl get pods -n "{{ namespace }}" -o wide | grep rep | grep "{{ node1.stdout }}" | awk '{print $1}'
```

Comment:
Can you please explain why this change is made? @giri can you please look at this task?

Reply:
Why did you remove the pv_name here?